src/allotropy/parsers/agilent_gen5/constants.py

```diff
@@ -39,6 +39,7 @@
 FILENAME_REGEX = r"^\d{6}_\d{6}_(?P<plate_identifier>.[^_]*)_.*\.txt$"
 
 NAN_EMISSION_EXCITATION = ["Full light", "Plug", "Hole"]
+NEGATIVE_ZERO = -0.0
```

Comment:
I think if we are going to use `-0.0` to represent a bad reading of a measurement this constant should be in `src/allotropy/parsers/constants.py` for consistency.

cc @james-leinas

Reply:
Yup, this makes sense. If we are ever going to apply to the other adapters, atleast for the ones for the new Plate Reader Schema.

Reply:
@joshua-benchling, agree here! We'll implement the constant NEGATIVE_ZERO for each of the adapters for these scenarios of required field in the measurement.